src/pages/result/fail/locales/zh-TW.ts

```diff
@@ -0,0 +1,10 @@
+export default {
+  'resultandfail.error.title': '提交失敗',
+  'resultandfail.error.description': '請核對並修改以下信息後，再重新提交。',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'resultandfail.error.description' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)